### DISCUSSION TITLE
Upgrade Minor Version from within the application?

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#3522] [General]
A quick questions.  I have been running FreshRss 1.17.1-dev (raspberry Pi) and doing updates along the way from within the web tool menu; thanks for the continued support and updates.  I did see an announcement that version 1.18 has been released.  However, when I request an update from within the application it says there are no updates.  Is there a different way I should upgrade from my older minor release version to this new release?  Today I run FreshRss on a Raspberry Pi (non docker).  Do I need to do a new pull of the latest code, or will the web interface update eventually push the newer code?  

![Screen Shot 2021-03-14 at 4 10 44 PM](https://user-images.githubusercontent.com/823352/111082869-ebe31280-84e0-11eb-9765-5c143b62b67e.png)

**Comment by JSB2000** [comment]
I'm in the same boat.  With the change from "Master" to "Edge," it appears that a LOT of links (including those which govern the update function) are no longer working.  When I click on Update, I get this:
![Update1](https://user-images.githubusercontent.com/8484306/111084164-66af2c00-84e7-11eb-9c64-b973d9a77ae1.png)

But when I click on Apply, I get this error:
![Update2](https://user-images.githubusercontent.com/8484306/111084189-821a3700-84e7-11eb-9414-5f832234efa9.png)

Suggestions? I'm sure we're not the only ones experiencing (or who will be experiencing) this.

Thanks!

**Reply by Alkarex** [reply]
@JSB2000 Could you please try again? I am in the process of changing a few things and just created a `master` tag for back-compatibility https://github.com/FreshRSS/FreshRSS/archive/master.zip

**Reply by JSB2000** [reply]
Bingo! That did the trick.  Thanks for the quick fix!

**Reply by marienfressinaud** [reply]
I also changed the URL on the update server, so the `master` tag should no longer required (at least for the update process)

**Comment by Alkarex** [comment]
@quella If you installed via ZIP, it will take some time for the new version to be visible (this is still a manual process, ping @marienfressinaud while waiting for a rework of our update process)

**Reply by quella** [reply]
Got it.  I will contact him on the process.  I recall installing via GIT, but it may have been the ZIP.

**Reply by marienfressinaud** [reply]
It should be good now @quella :)

**Reply by quella** [reply]
@marienfressinaud the published ZIP upgrade method in the Admin Guide is just what I was looking for.  Running 1.18.1-dev now and all seems to be functional.  Thanks again for the great support and assistance.

**Comment by quella** [comment]
This link may still be valid as well.  I will give it a shot in the AM using the .ZIP method.  With a backup prior.  :)

https://freshrss.github.io/FreshRSS/en/admins/04_Updating.html
